src/coreclr/jit/llvm.cpp

```diff
@@ -175,6 +213,70 @@ Value* buildAdd(llvm::IRBuilder<>& builder, GenTree* node, Value* op1, Value* op
     return mapTreeIdValue(node->gtTreeID, builder.CreateGEP(op1, op2));
 }
 
+FunctionType* getFunctionSignature(CORINFO_CONST_LOOKUP callTarget)
+{
+    return FunctionType::get(Type::getInt8PtrTy(_llvmContext), ArrayRef<Type*>(Type::getInt8PtrTy(_llvmContext)), false);
+}
+
+Value* genTreeAsLlvmType(GenTree* gt, Type* t)
+{
+    Value* v = getTreeIdValue(gt);
+    if (v->getType() == t)
+        return v;
+
+    if (gt->OperGet() == GT_CNS_INT && gt->gtType == var_types::TYP_INT)
+    {
+        return _builder->getInt({(unsigned int)t->getPrimitiveSizeInBits().getFixedSize(), (uint64_t)gt->AsIntCon()->gtIconVal, true});
+    }
+    failFunctionCompilation();
+}
+
+llvm::Value* buildUserFuncCall(GenTreeCall* gtCall, llvm::IRBuilder<>& builder)
+{
+    const char* symbolName = (*_getMangledSymbolName)(_thisPtr, gtCall->gtEntryPoint.handle);
+    // TODO: how to detect RuntimeImport ?
```

Comment:
A new JIT/EE interface flag or API

Reply:
I did this as a another method passed in via registerLlvmCallbacks.